en/docs/guides/identity-lifecycles/lock-account.md

```diff
@@ -1,69 +1,108 @@
 
-## Lock user accounts using the admin portal
+---
+
+## Configure the management console to enable account locking
 
-{insert-fragment}
+If you have not already configured WSO2 identity Server (WSO2 IS) for
+account locking, follow the instructions given below.
+
+{! fragments/enable-account-locking.md !}
+    
+---
+
+### Enable claims for account locking
+
+1.  Navigate to **Main** > **Identity** > **Claims** > **List** and select the `http://wso2.org/claims` claim dialect.  
+    For more information about claims, see [Configure Claims](../../../guides/dialects/configure-claims/).
+2.  Select the **Account Locked** claim and click **Edit**.  
+    ![account-locked-claim](../../../assets/img/guides/account-locked-claim.png) 
+3.  Select the **Supported by Default** check box and click **Update**
+.  
+    This is done to make the "Account Locked" status appear in the
+    user's profile.  
+    ![locked-status](../../../assets/img/guides/locked-status.png) 
+
+---
+
+## Lock user accounts using the management console
+
+1.  Navigate to **Main** > **Identity** > **Users and Roles** > **List** > **Users** and click on **User Profile** of the user you want to lock.
+2.  If it is the first time this particular account is being locked, a
+    text box will appear in front of the **Account Locked** field as
+    seen below.  
+    To lock the account, type true in the text box and click **Update**
+.  
+    ![admin-lock-account](../../../assets/img/guides/admin-lock-account.png)
+
+!!! note
+    If it is not the first time you are locking this user account, there
+    will be a check box instead of the text box (as shown above) in front of
+    the **Account Locked** field. Select the check box to lock the account
+    or deselect it to unlock the account and click **Update**.
 
 ---
 
 ## Lock user accounts using SCIM
 
-1. Open the `<IS-HOME>/repository/conf/scim2-schema-extension.config` file and add the following configuration. 
-
-   ```
-   {
-      "attributeURI":"urn:ietf:params:scim:schemas:extension:enterprise:2.0:User:accountLock",
-      "attributeName":"accountLock",
-      "dataType":"boolean",
-      "multiValued":"false",
-      "description":"Lock user account.",
-      "required":"false",
-      "caseExact":"false",
-      "mutability":"readwrite",
-      "returned":"default",
-      "uniqueness":"none",
-      "subAttributes":"null",
-      "canonicalValues":[],
-      "referenceTypes":[]
-   }
-   ```
-
-2. Then add the `accountLock` attribute as a sub-attribute of User.
-
-   ```
-   "subAttributes":"verifyEmail askPassword accountLock employeeNumber costCenter organization division department manager"
-   ```
+1.	Open the `<IS-HOME>/repository/conf/scim2-schema-extension.config` file and add the following configuration. 
```

Comment:
there is already an attribute called `accountLocked` in scim2-schema-extension.config file